unlock-app/src/components/interface/checkout/main/Renew.tsx

```diff
@@ -161,7 +156,11 @@ export function Renew({
   const onSign = async () => {
     setIsSigningMessage(true)
     try {
-      const signature = await signMessage(messageToSign!)
+      const walletService = await getWalletService()
+      const signature = await walletService.signMessage(
```

Comment:
Ha! We probably need to change this... (later PR!)

Reply:
But can you add a `TODO` please?

Reply:
It's already in TODO.